core-web/libs/dotcms-models/src/lib/dot-language.model.ts

```diff
@@ -7,4 +7,10 @@ export interface DotLanguage {
     defaultLanguage?: boolean;
     translated?: boolean;
     isoCode?: string;
+    variables?: { count: number; total: number };
+}
+
+export interface DotLanguagesISO {
+    countries: { code: string; name: string }[];
+    languages: { code: string; name: string }[];
```

Comment:
can we make `{ code: string; name: string }` an interface?

Reply:
done